README.md

```diff
@@ -2461,9 +2461,15 @@ AWS CodeGuru provides a number of features to help you improve the quality of yo
 
 ## Microsoft Azure Interview Questions
 
-1. What is Microsoft Azure, and how does it work?
-2. Explain the core services provided by Azure.
-3. What is Azure Resource Manager (ARM)?
+### 1. What is Microsoft Azure, and how does it work?
+Microsoft Azure is a cloud computing platform that provides a wide range of services, like storage, computing, and databases, hosted in Microsoft data centers. It lets businesses build, deploy, and manage applications globally, using tools and frameworks of their choice.
+
+### 2. Explain the core services provided by Azure.
+Azure provides core services like Compute, which includes virtual machines and containers; Storage for data solutions like Blob and Disk Storage; Networking to manage connectivity with services like Virtual Network; and Databases including SQL Database and Cosmos DB. These core services help build, deploy, and manage applications in the cloud efficiently
+
+### 3. What is Azure Resource Manager (ARM)?
+Azure Resource Manager (ARM) is the deployment and management service for Azure. It provides a way to organize resources, apply access control, and manage them through templates. ARM ensures resources are deployed consistently and makes it easy to monitor and control access across all resources in a solution.
```

Comment:
Azure Resource Manager (ARM) is not only a deployment and management service but also a RESTful API that allows users to programmatically manage their Azure resources.

Reply:
Correct answer has been added, so kindly approve this pull request.